python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -7,13 +7,16 @@
 
 import lastmile_utils.lib.core.api as core_utils
 import result
+import yaml
 from aiconfig.editor.server.server import run_backend_server
 from aiconfig.editor.server.server_utils import EditServerConfig, ServerMode
 from result import Err, Ok, Result
 
 
 class AIConfigCLIConfig(core_utils.Record):
     log_level: str | int = "WARNING"
+    config_path: str = "aiconfig_cli.config.yaml"
+    allow_usage_data_sharing: bool = False
```

Comment:
is this supposed to be used anywhere in this pr? I don't see it being referenced anywhere

Reply:
_process_cli_config()

Reply:
I'm confused